app/components/Account/index.js

```diff
@@ -18,7 +19,15 @@ class Account extends Component {
 
     return (
       <div>
-        <span>Address: {address}</span>
+        <div>Address: {address}</div>
+
+        <br />
```

Comment:
```<br />``` HTML element was deprecated in HTML 4 and obsolete in HTML 5. 
See: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br

I suggest to use css for styling it:
```
import classNames from 'classnames/bind'
import * as css from './index.css'
...
const cx = classNames.bind(css)
...
<div className={ cx('account') }>
    <div className={ cx('address') }>Address: {address}</div>
    <div className={ cx('network') }>Network: {config.networks[config.currentNetwork].url} </div>
</div>

//index.css
@import "./vars.scss"; <--- path to style constants
.address {
  marginBottom: @marginMd;  <--- Where marginMd should be a constant declared in the scss file (see above)
}
....
```

The idea behind of it is all components should use the same "styling", and take advantage of the css-in-modules practice. 

Let me know if you have questions :)